lib/datasource.js

```diff
@@ -335,14 +335,14 @@ DataSource.prototype.connect = function(callback) {
  * @returns {*}
  * @private
  */
-DataSource.prototype.setup = function(name, settings) {
+DataSource.prototype.setup = function(connectorName, settings) {
```

Comment:
The `name` argument is the `name` of the `dataSource`. It should be changed to `dsName` to make it obvious.